src/test/java/com/wikia/webdriver/testcases/discussions/CategoriesTests.java

```diff
@@ -72,15 +72,15 @@
    */
   @BeforeClass(groups = {DESKTOP})
   private void deleteCategoriesDesktop() {
-    deleteCategories(MercuryWikis.DISCUSSIONS_1);
+    deleteCategories(MobileWikis.DISCUSSIONS_1);
   }
 
   /**
    * Runs once before all tests in MOBILE group, deletes all categories on DISCUSSIONS_MOBILE wiki
    */
   @BeforeClass(groups = {MOBILE})
   private void deleteCategoriesMobile() {
-    deleteCategories(MercuryWikis.DISCUSSIONS_MOBILE);
+    deleteCategories(MobileWikis.DISCUSSIONS_MOBILE);
   }
 
   private void deleteCategories(String wikiName) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "deleteCategories" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
if this method is not used please remove it

Reply:
it is used